garrysmod/gamemodes/terrortown/gamemode/admin.lua

```diff
@@ -181,7 +181,7 @@ local ban_functions = {
    ulx    = ULib and ULib.kickban, -- has (ply, length, reason) signature
 
    evolve = function(p, l, r)
-               evolve:Ban(p:UniqueID(), l * 60, r) -- time in seconds
+               evolve:Ban(p:AccountID(), l * 60, r) -- time in seconds

```

Comment:
Evolve still expects a UniqueID as the first argument, so this won't work.

Reply:
Already being taken care of... As I said
> and some more are still awaiting to be patched but after this is merged (i.e. evolve's ban)

Reply:
Evolve isn't maintained anymore, though.